src/components/item_page/Item.test.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import Item from './Item';
+import { cleanup, render, waitForElement } from '@testing-library/react';
+import axios from '__mocks__/axios';
+
+describe('ItemsPage Test', () => {
+  beforeEach(() => {
+    jest.mock('axios');
```

Comment:
I see that you are wrapping `axios` in `utilities/fetch`, any reason you are mocking `axios` directly here?
https://8thlight.com/blog/eric-smith/2011/10/27/thats-not-yours.html
https://github.com/testdouble/contributing-tests/wiki/Don't-mock-what-you-don't-own

Reply:
We'll address this in PR #4